<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>368</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <layout class="QVBoxLayout" name="imgLayout"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListView" name="polygonList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addPolygonBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Añadir polígono</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addInCircleBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Añadir anillo interior</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cutBtn">
          <property name="text">
           <string>cortar</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="colorLayout" stretch="1,2,1,2,1,2,3">
          <item>
           <widget class="QLabel" name="labelR">
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="colorR">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelG">
            <property name="text">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="colorG">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelB">
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="colorB">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="setColorBtn">
            <property name="text">
             <string>Cambiar color</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="moveLayout" stretch="2,1,1,1,1">
          <item>
           <widget class="QLineEdit" name="moveDis">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveLeftBtn">
            <property name="text">
             <string>Mover Izq</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveRightBtn">
            <property name="text">
             <string>Mover Der</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveUpBtn">
            <property name="text">
             <string>Subir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="moveDownBtn">
            <property name="text">
             <string>Bajar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="rotateLayout">
          <item>
           <widget class="QLineEdit" name="rotateAngle">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotateLeftBtn">
            <property name="text">
             <string>Girar Izq</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotateRightBtn">
            <property name="text">
             <string>Girar Der</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="scaleLayout">
          <item>
           <widget class="QLineEdit" name="scaleRate">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scaleBtn">
            <property name="text">
             <string>Acercar / alejar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="flipLayout" stretch="1,1">
          <item>
           <widget class="QPushButton" name="horFlipBtn">
            <property name="text">
             <string>Flip horizontal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="verFlipBtn">
            <property name="text">
             <string>Flip vertical</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="otherLayout">
          <item>
           <widget class="QPushButton" name="showFillBtn">
            <property name="text">
             <string>Pantalla de relleno</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeBtn">
            <property name="text">
             <string>Eliminar polígono</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
